easybuild/easyconfigs/l/LibSoup/LibSoup-2.70.0-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'MesonNinja'
+
+name = 'LibSoup'
+version = '2.70.0'
+
+homepage = 'https://wiki.gnome.org/Projects/libsoup'
+description = """libsoup is an HTTP client/server library for GNOME. It
+uses GObjects and the glib main loop, to integrate well with GNOME
+applications, and also has a synchronous API, for use in threaded
+applications."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+toolchainopts = {'cstd': 'gnu89'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['54b020f74aefa438918d8e53cff62e2b1e59efe2de53e06b19a4b07b1f4d5342']
+
+builddependencies = [
+    ('binutils', '2.32'),
+    ('Meson', '0.51.2', '-Python-3.7.4'),
+    ('Ninja', '1.9.0'),
+    ('pkg-config', '0.29.2'),
+    ('GObject-Introspection', '1.63.1', '-Python-3.7.4'),
+    ('CMake', '3.15.3'),
+]
+
+dependencies = [
+    ('libxml2', '2.9.9'),
+    ('SQLite', '3.29.0'),
+    ('GLib', '2.62.0'),
+    ('libpsl', '0.21.0'),
+    ('cURL', '7.66.0'),
+]
+
+configopts = '-Dgssapi=disabled -Dvapi=disabled -Dtls_check=false '
```

Comment:
Just out of curiosity, what does `-Dtls_check=false` do?

Reply:
this disables support for TLS, as LibSoup only supports gnutls.
I've added a few comments for enabling it.

Reply:
Thanks for the explanation and adding the comments :+1: